src/views/setup/store.ts

```diff
@@ -97,6 +98,83 @@ export const useSetupStore = defineStore("setup", () => {
   let toolsFolder: Ref<string> = ref("");
   let toolsResults: Ref<IEspIdfTool[]> = ref([]);
   let extensionVersion: Ref<string> = ref("");
+  let idfPathError: Ref<string> = ref("");
+  let isInstallButtonDisabled: Ref<boolean> = ref(false);
+
+  function clearIdfPathError() {
+    idfPathError.value = "";
+    isInstallButtonDisabled.value = false;
+  }
+
+  function setIdfPathError(error: string) {
+    idfPathError.value = error;
+    isInstallButtonDisabled.value = !!error;
+  }
+
+  function validateEspIdfPath(path: string) {
+    clearIdfPathError();
+    vscode.postMessage({
+      command: "canAccessFile",
+      path,
+      pathIdfPy: `${path}/tools/idf.py`,
```

Comment:
Does `pathIdfPy` needs to be set here ? I think it could be generated in the vscode panel side. Why ? Well What about Windows-style paths `\\` ?